js/packages/components/main/List.tsx

```diff
@@ -331,8 +331,8 @@ const Conversations: React.FC<ConversationsProps> = ({ items, hasRequests }) =>
 							>
 								{t('main.messages.title')}
 							</Text>
-							{items.map((_) => {
-								return <ConversationsItem {..._} />
+							{items.map((_, key) => {
```

Comment:
you should avoid using an array index as key if you can, in this case you can use the conversation id I think